File: content/learn/arrays-easy/06-move-zeros-to-end/starter.js
function moveZerosToEnd(a) {
  // TODO: move all zeros to the end of a in place, preserving non-zero order.
}

const n = parseInt(readline(), 10);
const a = n > 0 ? readline().split(" ").map(Number) : [];
moveZerosToEnd(a);
print(a.join(" "));

File: content/learn/arrays-easy/06-move-zeros-to-end/starter.py
def move_zeros_to_end(a):
    # TODO: move all zeros to the end of a in place, preserving non-zero order.
    pass


n = int(input())
a = list(map(int, input().split())) if n > 0 else []
move_zeros_to_end(a)
print(" ".join(str(x) for x in a))
